apm/package.json

```diff
@@ -6,6 +6,6 @@
     "url": "https://github.com/atom/atom.git"
   },
   "dependencies": {
-    "atom-package-manager": "2.5.0"
+    "@atom-ide-community/atom-package-manager": "^2.5.0-atomic.2"
```

Comment:
Please use alias
```
npm i atom-package-manager@npm:@atom-ide-community/atom-package-manager
```

Reply:
Here is the full required command, getting lengthy :laughing: 

`npm i --global-style atom-package-manager@npm:@atom-ide-community/atom-package-manager`

and I frequently use --package-lock-only, so...

`npm i --global-style --package-lock-only atom-package-manager@npm:@atom-ide-community/atom-package-manager`

Talk about a keyboardfull. I'm okay with it. Just kind of funny.

Pushing it this way in a moment.

Reply:
Although we can just do  :unicorn: :sparkles: :keyboard: :rainbow: [`npm up`](https://docs.npmjs.com/cli-commands/update.html) :unicorn: :sparkles: :keyboard: :rainbow: next time.

Reply:
Just a technical note: package.json aliases require [npm 6.9.0](https://github.com/npm/cli/releases/tag/v6.9.0) and up.

(So it's handy that we're pinned to... exactly `npm@6.9.0` in CI already.)

That sets the system requirements to build our Atom at Node v10.12.0 and up, npm v6.9.0 and up. (And all the other stuff).

Reply:
Hmm, you can install with older npm.

If `node_modules` is already present from a `npm@>=6.9.0` install, then you can get various errors, such as:

<details><summary>Various error messages (click to expand):</summary>

```console
$ npm install
npm ERR! path /home/[user]/test-npm-6.5/node_modules/atom-package-manager/node_modules/npm
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '/home/[user]/test-npm-6.5/node_modules/atom-package-manager/node_modules/npm' -> '/home/[user]/test-npm-6.5/node_modules/atom-package-manager/node_modules/npm'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/[user]/.npm/_logs/2020-08-17T14_02_30_330Z-debug.log
```
or 

```console
>> Rebuilding apm dependencies with bundled Node v12.4.0 x64
./bin/npm: line 8: /home/[user]/test-npm-6.5/node_modules/atom-package-manager/bin/../node_modules/.bin/npm: No such file or directory
```
or

```console
$ npm i
npm ERR! Invalid dependency type requested: alias

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/[user]/.npm/_logs/2020-08-17T14_14_36_181Z-debug.log
```
</details>

If `node_modules` isn't present yet, you get this diff in `package-lock.json`:

```diff
8c8
<       "version": "npm:@atom-ide-community/atom-package-manager@2.5.0-atomic.2.1",
---
>       "version": "2.5.0-atomic.2.1",
```